types/index.test-d.ts

```diff
@@ -0,0 +1,33 @@
+import { IncomingMessage, ServerResponse, createServer } from "node:http";
+import { expectType } from "tsd";
+import finalhandler, { Options } from "..";
+
+const req: IncomingMessage = {} as IncomingMessage;
+const res: ServerResponse = {} as ServerResponse;
+
+const options: Options = {
+  env: "anEnv",
+  onerror: (err: any, req: IncomingMessage, res: ServerResponse): any => {
```

Comment:
```suggestion
  onerror: (err, req, res): any => {
```

`onerror` is typed within `Options`, so the types of its arguments are not required